Content.Server/GameTicking/Rules/PiratesRuleSystem.cs

```diff
@@ -39,6 +41,12 @@ public sealed class PiratesRuleSystem : GameRuleSystem<PiratesRuleComponent>
     [Dependency] private readonly PricingSystem _pricingSystem = default!;
     [Dependency] private readonly MapLoaderSystem _map = default!;
     [Dependency] private readonly NamingSystem _namingSystem = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+    /// <summary>
+    ///     Path to antagonist alert sound.
+    /// </summary>
+    private static string PirateAlert => "/Audio/Ambience/Antag/traitor_start.ogg"; //TODO make another sound
```

Comment:
Move to PiratesRuleComponent, also see if some gun or something serves as a better temp replacement (maybe a monkey scream?) Also is this even necessary? shouldnt Greet() handle this now?

Reply:
also it should be a SoundSpecifier not a path to a sound file

Reply:
Pirates don't have a role, so you can't use the added default functionalities.
I will try to find a replacement sound and will fix a SoundSpecifier.

Reply:
Made "/Audio/Effects/Diseases/monkey1.ogg" as an alert sound to greet new pirates.